src/schematools/contrib/django/migrations/0010_auto_20210330_1447.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.7 on 2021-03-30 14:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("datasets", "0009_auto_20210324_1157"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="datasettable",
+            name="db_table",
+            field=models.CharField(max_length=100),
+        ),
+    ]
```

Comment:
nitpick: You could place this in the other migration file, a bit silly to add 2 migration files in a single PR. preferably, the migration file has a clear name (`0009_add_versioning.py`)

Reply:
Oh, good one. Forgot to merge migrations.